app/main/posts/detail/detail.html

```diff
@@ -62,7 +71,7 @@ <h1 class="form-sheet-title survey-title">
 
         <post-value
           ng-repeat="(key,value) in post.values"
-          ng-if="form_attributes[key].type !== 'media' && isPostValue(key) && showType(form_attributes[key].type)"
+          ng-if="form_attributes[key].type !== 'media' && isPostValue(key) && showType(form_attributes[key].type) && form_attributes[key].input!=='tags'"
```

Comment:
these ifs are getting heavy, it would be nice to just define an array of the types not allowed.